SU2_CFD/include/solvers/CSpeciesSolver.hpp

```diff
@@ -100,7 +102,7 @@ class CSpeciesSolver : public CScalarSolver<CSpeciesVariable> {
    * \param[in] val_marker - Surface marker where the boundary condition is applied.
    */
   void BC_Inlet(CGeometry* geometry, CSolver** solver_container, CNumerics* conv_numerics, CNumerics* visc_numerics,
-                CConfig* config, unsigned short val_marker) final;
+                CConfig* config, unsigned short val_marker);
```

Comment:
Apply to others please, if you remove `final` it becomes `override` to still document that this is a virtual function that overrides the base class behavior
```suggestion
                CConfig* config, unsigned short val_marker) override;
```